app/code/Magento/InventoryInStorePickupApi/Api/Data/OperationResultInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryInStorePickupApi\Api\Data;
```

Comment:
Placement for this sort of "general" interfaces should be discussed. At this point it doesn't realy have anything to do with the Store Pickup